pkg/account/api/api_key_test.go

```diff
@@ -450,28 +444,40 @@ func TestGetAPIKeyMySQL(t *testing.T) {
 	}
 
 	patterns := []struct {
-		desc     string
-		setup    func(*AccountService)
-		req      *accountproto.GetAPIKeyRequest
-		expected error
+		desc           string
+		service        *AccountService
```

Comment:
We can create the service in the test loop. plz remove the same line in the other tests as well.

Reply:
Indeed, in the account package test, AccountV2ByEnvironmentID() can be mocked within setup(), so there is no need to separate services for each test, so I will try to fix it.
I will look into other packages to see if they are possible.